lua/astronvim/mappings.lua

```diff
@@ -245,6 +245,10 @@ if is_available "telescope.nvim" then
     function() require("telescope.builtin").git_bcommits { use_file_path = true } end,
     desc = "Git commits (current file)",
   }
+  maps.x["<leader>gc"] = {
```

Comment:
Mode `x` includes normal mode. So we should remove the `n` mode and make this just a single mapping. We also need to fix the messages to the the same

Reply:
Shouldn't `x` be only visual mode while `v` includes visual and selection mode? I'm not sure if the other mappings should apply to selection mode as well.